README.md

```diff
@@ -28,7 +28,7 @@ jobs:
           github_token: ${{ secrets.GH_PAT }}
 ```
 
-Note that the default `secrets.GITHUB_TOKEN` hasn't got [enough permissions][token-permissions], and cannot update the repository's topics. You should create a repo scoped [personal access token][pat] instead.
+Note that the default `secrets.GITHUB_TOKEN` hasn't got [enough permissions][token-permissions], and cannot update the repository's topics. You should create a repo scoped [personal access token][pat] instead. Alternativly, you can fork the repo and replace the action name to use your fork after reviewing the code to ensure security of your repository. This would mitigate the non-granular GitHub PAT permission scopes.
```

Comment:
```suggestion
Note that the default `secrets.GITHUB_TOKEN` hasn't got [enough permissions][token-permissions], and cannot update the repository's topics. You should create a repo scoped [personal access token][pat] instead. Alternatively, you can fork the repo and replace the action name to use your fork after reviewing the code to ensure security of your repository. This would mitigate the non-granular GitHub PAT permission scopes.
```